frontend/js/src/settings/Settings.tsx

```diff
@@ -111,33 +136,55 @@ export default function Settings() {
         </div>
 
         <div className="mb-15 donation-flairs-settings">
-          <div className="form-group">
+          <form className="form-group" onSubmit={submitFlairPreferences}>
             <h3>Flair Settings</h3>
             <p>
-              Choose which flair you want your username to show to let other
-              users know you donated. ListenBrainz.
+              Unlocked by <Link to="/donate/">donating</Link>. Some flairs are
+              only visible on hover.
             </p>
-            <select
-              id="flairs"
-              name="flairs"
-              className="form-control flair-select"
-              value={selectedFlair}
-              onChange={(e) => setSelectedFlair(e.target.value as Flair)}
-            >
-              {flairOptions.map((option) => (
-                <option key={option.value} value={option.value}>
-                  {option.label}
-                </option>
-              ))}
-            </select>
-            <button
-              onClick={submitFlairPreferences}
-              className="btn btn-lg btn-info"
-              type="button"
+            <div
+              className="flex flex-wrap"
+              style={{ gap: "1em", alignItems: "center" }}
             >
-              Submit
+              <div style={{ flexBasis: "300px" }}>
+                <Select
+                  id="flairs"
+                  name="flairs"
+                  isMulti={false}
+                  value={{
+                    value: selectedFlair,
+                    label: startCase(selectedFlair) as FlairName,
```

Comment:
Can we fix this case? I believe changing the className would be the easiest solution
![image](https://github.com/user-attachments/assets/2a43875e-53b2-4ac2-b9e8-d2f2dcc5e949)

Reply:
Also, selecting None makes it blank. Instead It should show the text "None"

![image](https://github.com/user-attachments/assets/cd3ffb5d-4e8a-461d-ad12-b763d0f43d2d)

Reply:
Good catch, these are showing the css class name instead of the label for them!